tests/e2e_tests/integration/07-saml.spec.ts

```diff
@@ -0,0 +1,151 @@
+import axios from 'axios';
+import * as fs from 'fs';
+import * as https from 'https';
+import jwtDecode from 'jwt-decode';
+
+import test, { expect } from '../fixtures/fixtures';
+
+test.describe('SAML Login', () => {
+  test.use({ storageState: 'storage.json' });
+  test.describe('SAML login via sso/id/login', () => {
+    test('Logs in using UI', async ({ environment, context, baseUrl, page, password, username }) => {
+      test.skip(environment !== 'staging');
+      const saml_settings={
+        'login':'morty',
+        'password':'panic',
+        'email':'msmith@samltest.id'
+      };
+      const timeout_ms=4000;
+      const storage=await context.storageState();
+      const jwt=storage['cookies'].find(cookie => cookie.name === 'JWT').value;
+      const metadata = await fs.readFileSync('fixtures/samltest.id_saml_idp.xml', 'utf8');
+
+      console.log(`running with ${baseUrl}`);
+      console.log(`IdP metadata len=${metadata.length} loaded and posting`);
+      const request = await axios({
+        url: `${baseUrl}api/management/v1/useradm/sso/idp/metadata`,
+        method: 'POST',
+        headers: {
+          Authorization: `Bearer ${jwt}`,
+          'Content-Type': 'application/samlmetadata+xml'
+        },
+        data: metadata,
+       httpsAgent: new https.Agent({
+          rejectUnauthorized: false
```

Comment:
this is running against staging, so I could remove the acceptance of anything :) but it also runs against local setups (i.e.: `TEST_ENVIRONMENT=localhost`) and without it it will not that easily so I vote: keep it.